### ISSUE TITLE
Spotless can't run Spotless on itself

### ISSUE THREAD

**Issue by nedtwigg** [diffplug/spotless#9]
For some reason, we can't add Spotless to the buildscript classpath of Spotless, which means we have to manually run [SelfTest.java](https://github.com/diffplug/spotless/blob/master/src/test/java/com/diffplug/gradle/spotless/SelfTest.java) from an IDE to check and apply formatting.

Here's the error, [straight from Travis](https://travis-ci.org/diffplug/spotless/builds/78004674#L163-L166).
```
* What went wrong:
A problem occurred configuring root project 'spotless'.
> Could not resolve all dependencies for configuration ':classpath'.
   > Module version com.diffplug.gradle.spotless:spotless:1.3.0-SNAPSHOT, configuration 'classpath' declares a dependency on configuration 'default' which is not declared in the module descriptor for com.diffplug.gradle.spotless:spotless:1.3.0-SNAPSHOT
```

To reproduce the issue, just checkout [`issueWithBootstrap`](https://github.com/diffplug/spotless/tree/issueWithBootstrap).  The commit that introduces the change is [c06839ae](https://github.com/diffplug/spotless/commit/c06839ae2fffafada6f375ee0a24b25e9d43e97c).

**Comment by nedtwigg** [comment]
[Possibly relevant Gist](https://gist.github.com/johnrengelman/9a20697b2246a9bfaca2).

**Comment by nedtwigg** [comment]
Hmm...  I only meant to reference this issue not close it.

**Comment by nedtwigg** [comment]
Fixed in 2.0.0-SNAPSHOT.
